graph-commons/src/main/scala/io/renku/graph/model/events.scala

```diff
@@ -67,6 +67,9 @@ object events {
     protected[this] override def maybeMax: Option[Instant] = instantNow.plus(24, HOURS).some
   }
 
+  final class ProjectDeliveryId private (val value: String) extends AnyVal with StringTinyType
```

Comment:
Is the `ProjectDeliveryId` still needed here?

Reply:
No I'll remove it